hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -75,8 +75,10 @@ public class DatanodeConfiguration {
   static final boolean CONTAINER_SCHEMA_V3_ENABLED_DEFAULT = false;
   static final long ROCKSDB_LOG_MAX_FILE_SIZE_BYTES_DEFAULT = 32 * 1024 * 1024;
   static final int ROCKSDB_LOG_MAX_FILE_NUM_DEFAULT = 64;
+  // one hour
   static final long ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_MICRO_SECONDS_DEFAULT =
-      6 * 60 * 60 * 1000 * 1000;
+      1 * 60 * 60 * 1000 * 1000;
+  static final int ROCKSDB_MAX_OPEN_FILES_DEFAULT = 1024;
```

Comment:
Is this working? looks like a numeric overflow case to me, Integer multiplication is being implicitly cast to long.
Tried a small assertion
```
    assertEquals(1L * 60 * 60 * 1000 * 1000, ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_MICRO_SECONDS_DEFAULT);

```
It indeed gives me a ``-694967296``

Secondly why are we changing the default value in the scope of adding a new property?

Reply:
@ayushtkn thanks, will check this.  

The hitchhiking is trying to avoid another small patch for this.